client/src/components/Tea/TeaList/TeaListContainer.jsx

```diff
@@ -1,36 +1,57 @@
 import React from "react";
 import { connect } from "react-redux";
+import { isEmpty } from "lodash";
 import { deleteTea, getTeas } from "../../../actions/teaActions";
 import TeaList from "./TeaList";
 
 export class TeaListContainer extends React.Component {
+  state = {
+    loading: true
+  };
+
   handleDeleteClick = tea => {
     this.props.handleDelete(tea);
   };
 
+  teaList = teasExist => {
+    if (isEmpty(teasExist)) {
```

Comment:
Not sure about the need for Lodash here. It would be easier to check whether `this.props.teas` contains `allTeas` and/or `teaIDs`.

Since you seem to have already removed the use of `lodash/merge` from your reducer, this means you could get rid of your Lodash dependency entirely.